WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -289,6 +299,17 @@ - (void)viewDidLoad
     [self refreshNavigationBarButtons:NO];
 }
 
+- (void)viewWillAppear:(BOOL)animated
+{
+    [super viewWillAppear:animated];
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(keyboardWasShown:)
+                                                 name:UIKeyboardDidShowNotification object:nil];
```

Comment:
Small detail: bring the `object:nil` parameter to the next line would improve readability.